components/eamxx/src/physics/ace/eamxx_ace_process_interface.cpp

```diff
@@ -0,0 +1,855 @@
+    - northward_wind_3
+    - northward_wind_4
+    - northward_wind_5
+    - northward_wind_6
+    - northward_wind_7
+    - LHTFLsfc // latent_heat_flux
+    - SHTFLsfc // sensible_heat_flux
+    - PRATEsfc // precipitation_rate
+    - ULWRFsfc // sfc_up_lw_radiative_flux
+    - ULWRFtoa // toa_up_lw_radiative_flux
+    - DLWRFsfc // sfc_down_lw_radiative_flux
+    - DSWRFsfc // sfc_down_sw_radiative_flux
+    - USWRFsfc // sfc_up_sw_radiative_flux
+    - USWRFtoa // toa_up_sw_radiative_flux
+    - tendency_of_total_water_path_due_to_advection
+  */
+
+  using KT  = KokkosTypes<DefaultDevice>;
+  using MT  = typename KT::MemberType;
+  using ESU = ekat::ExeSpaceUtils<typename KT::ExeSpace>;
+
+  auto output_view = m_output_field.get_view<const Real ****>();
+
+  auto T_mid = m_T_mid_ll.get_view<Real ***>();
+  auto qv = m_qv_ll.get_view<Real ***>();
+  auto wind = m_wind_ll.get_view<Real ****>();
+  auto p_mid = m_p_mid_ll.get_view<Real ***>();
+  auto p_int = m_p_int_ll.get_view<Real ***>();
+  auto dp_mid = m_dp_mid_ll.get_view<Real ***>();
+  auto phis = m_phis_ll.get_view<Real **>();
+  auto sfc_flux_dir_nir = m_sfc_flux_dir_nir.get_view<Real **>();
+  auto sfc_flux_dir_vis = m_sfc_flux_dir_vis.get_view<Real **>();
+  auto sfc_flux_dif_nir = m_sfc_flux_dif_nir.get_view<Real **>();
+  auto sfc_flux_dif_vis = m_sfc_flux_dif_vis.get_view<Real **>();
+  auto sfc_flux_sw_net = m_sfc_flux_sw_net.get_view<Real **>();
+  auto sfc_flux_lw_dn = m_sfc_flux_lw_dn.get_view<Real **>();
+  auto precip_liq_surf_mass = m_precip_liq_surf_mass.get_view<Real **>();
+  auto precip_ice_surf_mass = m_precip_ice_surf_mass.get_view<Real **>();
+
+  const auto num_lat = m_height;
+  const auto policy = ESU::get_default_team_policy(m_width, m_height);
+
+  // Populate the field views
+  // TODO: bypass this and just fill in the tensor directly?
+  // ALSO: note hackery and trickery!!!
+  Kokkos::parallel_for(
+      "ACE::postprocess", policy, KOKKOS_LAMBDA(const MT &team) {
+        const int ilon = team.league_rank();
+        Kokkos::parallel_for(
+            Kokkos::TeamVectorRange(team, num_lat), [&](const int &ilat) {
+              // TODO: is this even legit conceptually? Taking dp and p_int etc.
+              // TODO: second, copied these ak and bk values from an ai2/ace file
+              // TODO: formula is: p_i = ak_i + bk_i * PRESsfc
+              // std::map<int, std::pair<Real, Real>> m_ak_bk_values = {
+              //   {0, {64.247, 0.0}},
+              //   {1, {5167.14603, 0.0}},
+              //   {2, {12905.42546, 0.01755}},
+              //   {3, {13982.4677, 0.11746}},
+              //   {4, {12165.28766, 0.2896}},
+              //   {5, {8910.07678, 0.49806}},
+              //   {6, {4955.72632, 0.72625}},
+              //   {7, {2155.78385, 0.88192}}
+              // };
+              p_int(ilat, ilon, 8) = output_view(0, 0, ilat, ilon);
+              p_mid(ilat, ilon, 0) = p_int(ilat, ilon, 8) * 0.0 + 64.247;
```

Comment:
Do we need to fill in updated p and dp since they are not an input to ACE except for surface pressure?

Reply:
On this note, we should not allocate variables that ACE does not need, not sure how difficult that is.